src/main/res/values/strings.xml

```diff
@@ -21,4 +21,5 @@
     <string name="error_serialize_report">Failed to serialize bug report!</string>
     <string name="error_process_report">There was an error processing bug report!</string>
     <string name="error_save_screenshot">There was an error saving screenshot!</string>
+    <string name="summary_field_title">What happened?</string>
```

Comment:
@schukin I just noticed that there is a string called `what_happened_edit_text`. It's not used. Can you also append a commit to remove it?